moto/organizations/models.py

```diff
@@ -672,38 +674,39 @@ def list_targets_for_policy(self, **kwargs):
         ]
         return dict(Targets=objects)
 
-    def tag_resource(self, **kwargs):
-        account = next((a for a in self.accounts if a.id == kwargs["ResourceId"]), None)
-
-        if account is None:
+    def _get_resource_to_tag(self, resource_id):
+        if re.compile(utils.OU_ID_REGEX).match(resource_id) or re.match(
+            utils.ROOT_ID_REGEX, resource_id
+        ):
+            resource = next((a for a in self.ou if a.id == resource_id), None)
+        elif re.compile(utils.ACCOUNT_ID_REGEX).match(resource_id):
+            resource = next((a for a in self.accounts if a.id == resource_id), None)
+        elif re.compile(utils.POLICY_ID_REGEX).match(resource_id):
+            resource = next((a for a in self.policies if a.id == resource_id), None)
```

Comment:
This should have a test as well

Reply:
This is implicitly tested by the different tests of tagging the different resources. 

but I can add explicit tests if that is preferred?

Reply:
Yes, please. I can see it should work, but it's always good to have explicit tests

Reply:
I have added them.